### ISSUE TITLE
Weapons issue

### ISSUE THREAD

**Issue by FrozenDervish** [OwlcatOpenSource/WrathModificationTemplate#31]
When making new weapons it seems there is an issue.

1. They are not inheriting from type correctly.

And there may be other issues associated with making new weapons.

**Comment by BurlyWurly** [comment]
Yeah I made a custom weapon at one point and experienced this. The most notable thing was the animations. Wound up wielding glaives like a dagger or something. Adding the lines from type to the new weapon manually corrected it, but it'd be really nice not to have to do that.
